vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -227,8 +227,8 @@ bool FrameworkElementViewManager::UpdateProperty(
           transformMatrix.m44 = static_cast<float>(propertyValue[15].AsDouble());
 
           if (!element.IsLoaded()) {
-            element.Loaded([=](auto &&, auto &&) -> auto {
-              ApplyTransformMatrix(element, nodeToUpdate, transformMatrix);
+            element.Loaded([=](auto sender, auto&&) -> auto {
+              ApplyTransformMatrix(sender.as<xaml::UIElement>(), nodeToUpdate, transformMatrix);
```

Comment:
Heh, reference cycles are fun 🙂

As a sidenote, this is why implicit capture by value is evil...